pi-apps-terminal-bash-edition.sh

```diff
@@ -213,41 +166,100 @@ while [ "$1" != "" ]; do
 			exit $?
 		;;
 		list-all)
-			#list all the apps
-			list-all
+			shift
+			case $1 in
+				-d|--description) #print with descriptions
+					#list all the apps
+					IFS=$'\n'
+					for app in $(list_apps cpu_installable); do
+					     echo -e "\n${bold}${inverted}${light_blue}$app${normal}"
+					     echo -e "${green}$(cat "$DIRECTORY/apps/$app/description")${normal}"
+					done	
+				;;
+				*)
+					list_apps cpu_installable
```

Comment:
Could you add the colors here as well please?

Reply:
Added in https://github.com/cycool29/PiApps-terminal_bash-edition/commit/71032299d83b4ef771db775643593a7a67f42dc9.